src/components/Articles.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+import { getArticles } from '../utils/api';
+
+const Articles = () => {
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		getArticles().then((articlesFromApi) => {
+			console.log(articlesFromApi.articles);
+			setArticles(articlesFromApi.articles);
```

Comment:
Not an urgent thing! But with this getArticles function in the .then we're saying that we get back 'articlesFromApi' but then we are accessing the 'articles' property. I'd maybe rename articlesFromApi to just be 'response' or you could even destructure this to obtain the articles directly by having it as getArticles().then(({articles})) => { setArticles(articles) }. But not urgent!!